### PR TITLE
Updating HACKING.md and fixing up Makefile

### PR DESCRIPTION
**Pull request by ewolinetz**

### CODE REVIEW

**Comment by ewolinetz** [comment]
/hold
i think i missed initializing open distro.. i saw a `SG11` error when curling that i missed

File: HACKING.md [hunk 1]
```diff
@@ -32,10 +32,59 @@ curl -kivX GET -H "Authorization: Bearer $(oc whoami -t)" https://localhost:6000
...
     cp elasticsearch/sgconfig/roles_mapping.yml ~/tmp/elasticsearch-6.8.1/plugins/opendistro_security/securityconfig/roles_mapping.yml
     ```
 4. Start ES and verify curl works `curl -kivX GET  --cert config/kirk.pem --key config/kirk-key.pem --cacert config/root-ca.pem  https://localhost:9200`
 5. Copy certificates to `__output` directory to be used by proxy `make copy-es-certs`
 6. Start proxy
+
+
+## Installing, configuring, and starting local instance of Elasticsearch and Proxy
+1. Download ES
+  `curl https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.6.2.rpm -o elasticsearch-6.6.2.rpm`
```
Comment (jcantrill) [inline HACKING.md]: RPM instead of zip?

Comment (ewolinetz) [reply HACKING.md]: that's what I used... this way we get the service and all that set up for us...

Comment (ewolinetz) [reply HACKING.md]: @jcantrill is the a priority to ensure we can install this on any system? if so i can rework this using a `.zip` instead

Comment (jcantrill) [reply HACKING.md]: The priority is only to quick setup for hacking the proxy.  My expectation is that either zip or tar.gz that locates ES in a singular directory is preferred to rpm which scatters pieces everywhere.  We are advising how to hack the proxy, not necessarily setup an ideal ES installation

Comment (ewolinetz) [reply HACKING.md]: this isn't necessarily setting up an ideal ES installation... its a very quick way to get it up and running for hacking on the proxy.. i'll poke around with using a different means of starting up ES...

**Comment by ewolinetz** [comment]
@jcantrill updated `HACKING.md` to use the `.tar.gz` file instead

File: HACKING.md (hunk 1 above)
Comment (jcantrill) [reply HACKING.md]: Alternate is not high priority for me if you want to merge this and we can alter later

**Pull request merged**